lib/wx/doc/src/gl.xml

```diff
@@ -1250,15 +1250,15 @@
   </func>
   <func>
     <name name="drawTransformFeedbackStream" arity="3" clause_i="1" since=""/>
```

Comment:
This file is generated from the official opengl docs do not edit (and also doesn't build after your changes)